specification/hdinsight/resource-manager/readme.md

```diff
@@ -110,6 +110,12 @@ directive:
     reason: Update the default error response to a new format would be a braking change for service.
 ```
 
+```yaml
+directive:
```

Comment:
Please remove this code. It doesn't work per [this doc](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/85/Swagger-Suppression-Process?anchor=obtaining-suppression-for-avocado) and it blocks your PR from being merged.

Reply:
removed, thanks @konrad-jamrozik for your help.